app/api/v1/endpoints/users.py

```
@@ -0,0 +1,34 @@
+from ....crud import user as user_crud
+from ....db.session import get_db
+from typing import List
+from loguru import logger
+
+router = APIRouter()
+
+@router.get("/", response_model=List[user_schema.UserCreate])
+async def read_users(
+    skip: int = 0,
+    limit: int = 100,
+    db: Session = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)